src/client/client.cpp

```diff
@@ -103,6 +103,7 @@ Client::Client(
 		GameUI *game_ui,
 		ELoginRegister allow_login_or_register
 ):
+	m_allow_login_or_register(allow_login_or_register),
```

Comment:
This variable should be private. There are no accesses outside of the Client class.